repo/tlv/definitions.go

```diff
@@ -0,0 +1,50 @@
+//go:generate gondn_tlv_gen
+package tlv
+
+import (
+	enc "github.com/named-data/ndnd/std/encoding"
+	spec "github.com/named-data/ndnd/std/ndn/spec_2022"
+)
+
+var SyncProtocolSvsV3 = enc.Name{
+	enc.NewKeywordComponent("ndn"),
+	enc.NewKeywordComponent("svs"),
+	enc.NewVersionComponent(3),
+}
+
+type RepoCmd struct {
```

Comment:
Do you want to modify `ndn-python-repo` spec or create a new spec doc?

Reply:
Yes, good point. This spec is a bit ad-hoc at the moment because SVS-ALO itself isn't specified.